drdown/notifications/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
```

Comment:
Não se se a ferramenta de análise estática reclama de arquivos que não fazem nada, se for só deleta ^^